ui/StatusQ/src/StatusQ/Components/StatusUserImage.qml

```diff
@@ -16,41 +11,56 @@ Loader {
     property int imageWidth: 44
 
     property string name
+    property bool usesDefaultName: false
     property string image
     property bool showRing: !ensVerified && !root.isBridgedAccount
     property bool interactive: true
     property bool disabled: false
     property bool ensVerified: false
     property bool loading: false
     property bool isBridgedAccount: false
-    property int onlineStatus: Constants.onlineStatus.unknown
+    // TODO replace this with booleans since we do not have access to Constants
+    property int onlineStatus: -1
 
-    property int colorId
```

Comment:
We could probably keep the `colorId` and have it converted to the actual `userColor` as an implementation detail, and move `Utils.colorForColorId` to StatusQ.Utils, wdyt? It would simplify the usage of this component a lot

Reply:
I thought it would use the backend, but it seems like it doesn't.

However, I wonder if it's not cleaner to leave it as a color in the optic of it being used later by other apps? `colorId` is very tied to our own project

Reply:
Yeah, probably cleaner to have a `property color` on the component level